src/ui/public/kuery/errors/index.js

```diff
@@ -0,0 +1,10 @@
+import { KbnError } from '../../errors';
```

Comment:
nit: This is the first KbnError that's not just in `/src/ui/public/errors.js`, so the argument can be made to just put this in there as well.

Reply:
I think the difference here is that these errors are only meant to be thrown by the parser, so it makes sense to co-locate them with the grammar. Eventually I imagine us having a bunch of these parser errors. I wouldn't have even extended `KbnError` if extending Error was easier in javascript.

In general I've been trying to keep all kuery code inside the kuery directory so it doesn't become an incomprehensible mess spread across the entire codebase (see: visualize).